microsoft-edge/extensions-chromium/publish/api/addons-api-reference.md

```diff
@@ -1,20 +1,20 @@
 ---
-title: Microsoft Edge Add-ons API Reference (in private preview)
+title: Microsoft Edge Add-ons API Reference (is currently in beta)
```

Comment:
please change "title:" yaml field x2, titles x2, link text in all .md files that point to these articles, and the TOC titles x2, from (is currently in beta) to simply (Beta)